bin/admin/bootstrap-cluster.sh

```diff
@@ -72,8 +149,8 @@ build_nfs_server() {
   echo "building a NFS server...done!"
 }
 
-usage="Usage : $0 <node_type>"
-usage="$usage\n node_type: master, client, nfs_server"
+usage="Usage : $0 <node_type>\n"
```

Comment:
This change is used for ``printf`` to has a better looking of the output format.